internal-packages/ai/src/tools/check-math-with-mathjs/index.ts

```diff
@@ -111,8 +111,20 @@ export class CheckMathWithMathJsTool extends Tool<CheckMathAgenticInput, CheckMa
     version: '2.0.0',
     category: 'analysis' as const,
     costEstimate: '~$0.02-0.05 per statement (uses Claude with multiple tool calls)',
-    path: '/tools/check-math-with-mathjs',
-    status: 'stable' as const
+    path: '/api/tools/check-math-with-mathjs',
```

Comment:
Why do these need to store the path? Isn't the path consistent between tools? If not, maybe we should focus on making sure it's consistent.